en-US/Players_Guide/03_Creating_a_Character/Classes/Adept/Adept.md

```diff
@@ -1,4 +1,234 @@
 #### Adept
+<!-- spell-checker:words Arkylie Kilyle nerfed -->
+
+By honing their awareness beyond ordinary limits, adepts unlock hidden potential within themselves, achieving feats that others consider impossible.
+
+##### Class Features
+
+As an Adept, you gain the following class features.
+
+###### Hit Points
+
+**Hit Dice:**
+`1d8` per Adept level
+\
+**Hit Points at 1st Level:**
+`8 + your Constitution modifier`
+\
+**Hit Points at Higher Levels:**
+`1d8 (or 5) + your Constitution modifier per Adept level after 1st`
+
+###### Proficiencies
+
+**Armor:**
+None
+\
+**Weapons:**
+Simple weapons, shortswords
+\
+**Tools:**
+Choose one type of artisan’s tools or one musical instrument
+\
+**Saving Throws:**
+Strength, Dexterity
+\
+**Skills:**
+Choose two from Acrobatics, Athletics, History, Insight, Religion, and Stealth
+
+###### Equipment
+
+You start with the following equipment, in addition to the equipment granted by your background:
+- (a) a shortsword or (b) any simple weapon
+- (a) a dungeoneer’s pack or (b) an explorer’s pack
+- 10 darts
+
+###### The Adept (table) {#Adept_the_adept_table}
+
+| Level | Proficiency Bonus | Martial Arts | Focus Points | Unarmored Movement | Features                                     |
```

Comment:
In the Google Doc, Arkylie (Kilyle) suggested providing a formula for focus points and unarmored movement in a sidebar instead.
I ket it as is, at least for now.